prow/pod-utilities.md

```diff
@@ -64,3 +83,38 @@ Additional fields may be required for some use cases:
 to clone the repo to a path different than the default of `/go/src/github.com/org/repo/` (e.g. `/go/src/k8s.io/kubernetes/kubernetes`).
 - Jobs that require additional repos to be checked out can arrange for that with
 the `exta_refs` field.
+
+```yaml
+- name: post-job
+  agent: kubernetes
+  decorate: true
+  ssh_key_secrets:
+  - ssh-secret
+  clone_uri: "git@github.com:org/repo.git"
```

Comment:
Might want to name this something more goofy like `paranoid-parrot/skunkworks-project` (or maybe something in between)

Reply:
Switched to use the golang template struct that is available. This should now be the same for all private repo use cases and should also be clearer.